core/src/main/scala/com/microsoft/azure/synapse/ml/causal/DoubleMLEstimator.scala

```diff
@@ -256,7 +244,7 @@ class DoubleMLEstimator(override val uid: String)
       .setFeaturesCol(treatmentResidualVecCol)
       .setFamily("gaussian")
       .setLink("identity")
-      .setFitIntercept(false)
+      .setFitIntercept(true)
```

Comment:
would this model benefit from exposing all of these as params with these defaults?

Reply:
@sarahshy

Reply:
I think we should keep the parameters fixed and hidden since there won't (at least, shouldn't) be a case where they need to be changed.